src/diffusers/utils/hub_utils.py

```diff
@@ -94,14 +94,14 @@ def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
 
 
 def load_or_create_model_card(
-    repo_id_or_path: Optional[str] = None, token: Optional[str] = None, is_pipeline: bool = False
+    repo_id_or_path: Optional[str] = None, token: Optional[str] = None, is_pipeline: bool = False, **kwargs
```

Comment:
```suggestion
    repo_id_or_path: Optional[str] = None, token: Optional[str] = None, is_pipeline: bool = False, model_description: Optional[str] = None
```

Let's try to avoid `kwargs` in function signatures 🙏